cmd/internal/client/push.go

```diff
@@ -10,9 +12,9 @@ var ()
 
 // CmdPush implements the carrier orgs command
 var CmdPush = &cobra.Command{
-	Use:   "push NAME",
+	Use:   "push NAME [PATH_TO_APPLICATION_SOURCES]",
 	Short: "Push an application from the current working directory",
```

Comment:
+ `... or the path passed as argument"

Reply:
Yes, definitely. Will fix.

Reply:
Done with commit 28ff611d2058a50b69563b5e9285968ec205c187